awsenv

```diff
@@ -67,7 +67,7 @@ credentials_cache_not_expired() {
   # treat empty cache
   [ -n "$(jq < "$cache")" ] || return 1
 
-  jq --exit-status '(.Credentials.Expiration | fromdate) > now' < "$cache" > /dev/null
+  jq --exit-status '(.Credentials.Expiration | sub("\\+.*$"; "Z") | fromdate) > now' < "$cache" > /dev/null
```

Comment:
I think that awsenv shouldn't substitute timezone string to `Z` if non-utc timezones.


```suggestion
  jq --exit-status '(.Credentials.Expiration | sub("\\+00:00$"; "Z") | fromdate) > now' < "$cache" > /dev/null
```


It will show jq error if aws uses a local timezone in a future.
Do you think above suggestion?

Reply:
This should be safer. Sry for my poor consideration.
I've f-pushed the change.